public/indexAdmin.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>AMA App</title>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <style>
+	.container-fluid {
+    width: 100%;
+    padding-right: 0px;
+    padding-left: 0px;}
+	.navbar-header{
+	width:70%;
+	}
+	#bs-example-navbar-collapse-1{
+	width:30%;
+	}
+	li {
+    float: left;
+    padding-left: 10px;
+}
+.col-sm-2{
+background: cadetblue;
+padding:5px;
+}
+.col-sm-8{
+padding:60px;
+}
+.row{
+    margin-right: 0px;
+    margin-left: 0px;
+	}
+a, a:active, a:hover {
```

Comment:
You should use the `.sidebar` class here as well, because this will make all links on the admin white. If they appear in the text, they won't be visible on the white background.

1. Replace this part with `.sidebar a`.
2. It's a good practice to have a different style for hover/active - that identifies a link. Maybe you can underline the text on hover? See below.
```
.sidebar a:hover, .sidebar a:active {
    text-decoration: underline;
}
```